Source/Core/Core/HW/DSP.cpp

```diff
@@ -212,6 +212,7 @@ void EnableInstantDMA()
 	CoreTiming::RemoveEvent(et_CompleteARAM);
 	CompleteARAM(0, 0);
 	instant_dma = true;
+	ERROR_LOG(DSPINTERFACE, "Enabling Instant ARAM DMA hack");
```

Comment:
gotta point out that most users who would get confused by the logs are probably not going to be dicking around in them anyhow.

Just because I don't know enough to program here doesn't mean a log like this would confuse me.

I vote to push information like this for the more technically inclined.
for the rest, if they cannot use google, they honestly need to learn how.